### PR TITLE
[OREE-2123] Added test for presence of OutreachContext.email

### PR DESCRIPTION
**Pull request by palootcenas-outreach**
# [OREE-2123]

Added test for presence of OutreachContext.email
A follow up of #104 

[OREE-2123]: https://outreach-io.atlassian.net/browse/OREE-2123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/index.test.ts [hunk 1]
```diff
@@ -45,12 +57,38 @@ describe('sdk tests', () => {
...
 
+      window.clearTimeout = originalClearTimeout;
       window.setTimeout = originalTimeout;
       window.addEventListener = originalEventListener;
       oauthService.openPopup = originalOpenPopup;
       jest.restoreAllMocks();
     });
 
+    describe('Outreachcontext initialized', () => {
+      it('will resolve context promise when success', async () => {
```
Comment (honeydripper) [inline tests/index.test.ts:68]: Can we change this to something that describes the test better? Like: "Supported email context parameters are populated"

File: tests/index.test.ts [hunk 2]
```diff
@@ -45,12 +57,38 @@ describe('sdk tests', () => {
...
+        var message = new InitMessage();
+        message.result = '200';
+        message.application = new Application();
+        message.application.store = new ManifestStore();
+        message.context = [
```
Comment (honeydripper) [inline tests/index.test.ts:73]: can we include all parameters that are described here? https://developers.outreach.io/client-extensions/javascript-sdk/#email 
Also the docs description of email context parameters `subj` and `to` does not correspond to the implementation. Please fix either here or in the docs (preferably).

**Review by honeydripper — CHANGES_REQUESTED** [review]

File: tests/index.test.ts (hunk 2 above)
Comment (palootcenas-outreach) [reply tests/index.test.ts:73]: Yes.
I will fix the docs. `To` should be an array in docs. Subject states "message subject", what's wrong about that?

Comment (honeydripper) [reply tests/index.test.ts:73]: In the docs it is called `subj` and the key name on the context object according to the test is `subject`

**Review by honeydripper — CHANGES_REQUESTED** [review]

File: tests/index.test.ts (hunk 2 above)
Comment (palootcenas-outreach) [reply tests/index.test.ts:73]: That is correct. In `OutreachContext` we use full names. In query params we use `eml.subj`

Comment (honeydripper) [reply tests/index.test.ts:73]: But the docs are about OutreachContext object and not about the query parameters, so I guess we need to fix the docs.

**Review by honeydripper — APPROVED** [review]

**Review by vSleichert — APPROVED** [review]

**Pull request merged**